changes/ee/fix-16032.en.md

```diff
@@ -0,0 +1 @@
+Fixed an issue where, if a node was restarted and had just recorded a metric sample (e.g.: dropped messages) containing positive values, it could record a negative value when it took its next sample.
```

Comment:
maybe make it more clear about what user may observe previously in the dashboard or in the logs